src/views/Dashboard/Lab/result.vue

```diff
@@ -121,6 +121,7 @@ import sendTransaction from '../../../lib/send-transaction'
 import localStorage from '../../../lib/local-storage'
 import Wallet from '../../../lib/dgnx-wallet'
 import { mapState } from 'vuex'
+import { SUCCESS } from '@/constants/specimen-status'
```

Comment:
why only 1 status success?

Reply:
this is for dissabled button if speciment status is success.